Sources/MockoloFramework/Models/MethodModel.swift

```diff
@@ -130,18 +182,23 @@ final class MethodModel: Model {
         let postfix = diff > 0 ? String(diff) : self.signatureComponents[level - 1]
         return name(by: level - 1) + postfix
     }
-
     
     func render(with identifier: String, typeKeys: [String: String]? = nil) -> String? {
         if processed {
             if isInitializer {
                 return nil
             }
-            
-            let ret = self.data.toString(offset: offset, length: length)
-            return ret
+
+            if let modelDescription = modelDescription {
```

Comment:
This whole block is just `return modelDescription ?? self.data?.toString(offset: offset, length: length)`